apps/rush-lib/src/data/test/repo/rush.json

```diff
@@ -17,6 +17,12 @@
     "sampleEmail": "mrexample@contoso.com"
   },
 
+  "rushHooks": {
+    "postBuild": [
```

Comment:
Be sure to document this in the schema

---
In reply to: [117142380](https://github.com/Microsoft/web-build-tools/pull/196#discussion_r117142380) [](ancestors = 117142380)